pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -0,0 +1,629 @@
+// api.pb.go could be generate by hack/update-generate-runtime.sh
+syntax = 'proto2';
```

Comment:
Some concerns when switching from proto2 to proto3 is that proto3 removes `optional` `required` keywords. And everything becomes `optional`.